### ISSUE TITLE
(GitHub code show whitespace) show all whitespaces

### ISSUE THREAD

**Issue by jerone** [Mottie/GitHub-userscripts#27]
I tried [GitHub code show whitespace](https://github.com/Mottie/GitHub-userscripts/wiki/GitHub-code-show-whitespace) today, and found that it only shows leading and trailing spaces & tabs. There is [another script](https://github.com/glebm/render-whitespace-on-github) that does show the inline whitespaces, and I was wondering if you can implement this too.

**Comment by Mottie** [comment]
Hi @jerone!

That sounds like a good idea, I'll take a look!

**Comment by jerone** [comment]
@Mottie Saw you commit, all inline whitespaces work perfectly now. 🎉 

I do however see an extra space character before every line now. For example https://github.com/jerone/UserScripts/commit/0c1e262e7ac6158b9f188f8b23f57ed2b0e2d75a, the first userscript block in GitHub_Commit_Compare.user.js is not prefixed with a space in the code, still I see the space character.
![2017-10-02 10_38_44-merge pull request 120 from jerone_github-commit-compare jerone_userscripts 0](https://user-images.githubusercontent.com/55841/31069863-82ff8aea-a75e-11e7-8894-bce740230430.png)

Hope you can reproduce and find a solution.

**Comment by Mottie** [comment]
Hmm, the space you're reporting, I think, is due to the replacement of the "+" (or "-") in the diff. So the script that is replacing the that symbol is the culprit, it's the `github-remove-diff-signs.user.js` script, if you're using it.

I do however see an issue in the readme file from that same commit - it looks like the space inside the span is being removed and the carriage returns are misaligned. I'll try to work on this later today.

**Comment by Mottie** [comment]
> Maybe you can use a different whitespace character for that...

I tried that, but I think it's best to completely remove the "+", "-" or first space because the whole reason we're doing it is to make it easier to copy/paste the code. The main issue then becomes removing the leading space in the expanded code section. I'll leave the detection and processing of the expanded section for a later version.

> It's not showing any whitespace character.

It's the weirdest thing ever... I had to switch all the spans to use `display: inline` to get them to show, then I found that *only* the diffs for markdown and non-extension files (e.g. LICENSE) were effected by a misalignment of the placeholder symbols.

I ended up adding a work-around to adjust the specific diff files and then removing all tab-size adjustment code since I couldn't set a width with `display: inline`. It should all be working smoothly now.
